src/app/code-set/code-set-details/code-set-details.component.ts

```diff
@@ -38,225 +29,135 @@ import { BroadcastService } from '@mdm/services/broadcast.service';
-    this.CodeSetDetails();
+     this.codeSetDetails();
   }
 
-  public showAddElementToMarkdown() { // Remove from here & put in markdown
+  public showAddElementToMarkdown() {
+    // Remove from here & put in markdown
     this.elementDialogueService.open('Search_Help', 'left' as DialogPosition);
   }
 
   ngOnInit() {
-    this.editableForm = new EditableDataModel();
-    this.editableForm.visible = false;
-    this.editableForm.deletePending = false;
-
-
-    this.editableForm.show = () => {
-      this.editForm.forEach(x => x.edit({
-        editing: true,
-        focus: x.name === 'moduleName' ? true : false
-      }));
-      this.editableForm.visible = true;
-    };
-
-    this.editableForm.cancel = () => {
-      this.editingService.stop();
-      this.editForm.forEach(x => x.edit({ editing: false }));
-      this.editableForm.visible = false;
-      this.editableForm.validationError = false;
-      this.errorMessage = '';
-      this.editableForm.description = this.result.description;
-      if (this.result.classifiers) {
-        this.result.classifiers.forEach(item => {
-          this.editableForm.classifiers.push(item);
-        });
+    this.subscription = this.messageService.changeUserGroupAccess.subscribe(
+      (message: boolean) => {
+        this.showSecuritySection = message;
       }
-      if (this.result.aliases) {
-        this.result.aliases.forEach(item => {
-          this.editableForm.aliases.push(item);
-        });
-      }
-
-    };
-
-    this.subscription = this.messageService.changeUserGroupAccess.subscribe((message: boolean) => {
-      this.showSecuritySection = message;
-    });
+    );
   }
 
-  CodeSetDetails(): any {
-
-    this.subscription = this.messageService.dataChanged$.subscribe(serverResult => {
-
-      if(serverResult.domainType !== 'CodeSet')
-      {
-        return;
-      }
-
-      this.result = serverResult;
-
-
-      this.getModelGraph(this.result.id);
-
-      this.editableForm.description = this.result.description;
-      if (this.result.classifiers) {
-        this.result.classifiers.forEach(item => {
-          this.editableForm.classifiers.push(item);
-        });
-      }
-      if (this.result.aliases) {
-        this.result.aliases.forEach(item => {
-          this.editableForm.aliases.push(item);
-        });
-      }
-      if (this.result.semanticLinks) {
-        this.result.semanticLinks.forEach(link => {
-          if (link.linkType === 'New Version Of') {
-            this.compareToList.push(link.target);
-          }
-        });
-      }
-
-      if (this.result.semanticLinks) {
-        this.result.semanticLinks.forEach(link => {
-          if (link.linkType === 'Superseded By') {
-            this.compareToList.push(link.target);
-          }
-        });
-      }
-
-      if (this.result != null) {
-        this.hasResult = true;
-        this.watchDataModelObject();
+  codeSetDetails(): any {
+    this.subscription = this.messageService.dataChanged$.subscribe(
```

Comment:
Same issue with this, using `@Input()` would be much simpler if the data just comes from the parent component.

Defer to another clean up issue.